pkg/action/bootentries.go

```diff
@@ -133,13 +133,9 @@ func selectBootEntrySystemd(cfg *config.Config, entry string) error {
 	if err != nil {
 		return err
 	}
-	assessment, err := utils.ReadAssessmentFromEntry(cfg.Fs, bootFileName, cfg.Logger)
```

Comment:
I think this function is only used here and the tests so maybe it can be removed